doc/src/quickstart/index.rst

```diff
@@ -485,6 +458,68 @@ which we can visualize with:
         --route_chan_width 100 \
         --analysis --disp on
 
+Manually Running VTR with ODIN II
+----------------------------------
+Let's start by making a new directory for us to work in:
```

Comment:
Add (at the very start of this section):

VTR includes a second synthesis tool, ODIN II.  Below we explain how to run this alternative synthesis. flow.